spec/models/job_post_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.